src/main/java/de/rabea/Game/HumanPlayer.java

```diff
@@ -0,0 +1,45 @@
+package de.rabea.game;
+
+import de.rabea.ui.UserInterface;
+
+public class HumanPlayer implements Player {
+    UserInterface userInterface;
+    Mark mark;
+
+    public HumanPlayer(UserInterface userInterface, Mark mark) {
+        this.userInterface = userInterface;
+        this.mark = mark;
+    }
+
+    public Mark mark() {
+        return mark;
+    }
+
+    public int getPosition(Board board) {
+        userInterface.askForPosition();
+        return returnUsersPosition(board);
+    }
+
+    public int returnUsersPosition(Board board) {
+        String userInput = userInterface.readUserInput();
```

Comment:
There are quite a lot of calls to userInterface here. Could be feature envy?  http://programmers.stackexchange.com/questions/212128/what-is-a-feature-envy-code-and-why-is-it-considered-a-code-smell